docs: integrations sidebar updates (#262)
langchain-ai/docs: src/oss/integrations/providers/index.mdx

| [Anchorbrowser](anchor_browser) | [langchain-anchorbrowser](https://pypi.org/project/langchain-anchorbrowser/) | ![Downloads](https://static.pepy.tech/badge/langchain-anchorbrowser/month) | ![PyPI - Version](https://img.shields.io/pypi/v/langchain-anchorbrowser?style=flat-square&label=%20&color=orange) | ❌ |
| [MCP Toolbox](/docs/integrations/providers/toolbox-langchain/) | [toolbox-langchain](https://pypi.org/project/toolbox-langchain/) | ![Downloads](https://static.pepy.tech/badge/toolbox-langchain/month) | ![PyPI - Version](https://img.shields.io/pypi/v/toolbox-langchain?style=flat-square&label=%20&color=orange) | ❌ |
| [Scrapeless](/docs/integrations/providers/scrapeless/) | [langchain-scrapeless](https://pypi.org/project/langchain-scrapeless/) | ![Downloads](https://static.pepy.tech/badge/langchain-scrapeless/month) | ![PyPI - Version](https://img.shields.io/pypi/v/langchain-scrapeless?style=flat-square&label=%20&color=orange) | ❌ |

## All Providers

> Click [here](/oss/python/integrations/providers/all_providers) to see all providers. Or search for a
> provider using the Search field in the top-right corner of the screen.

Reviewer: ```suggestion
You can view [all providers](/oss/python/integrations/providers/all_providers) or search for a provider using the search field.
```
